src/udiskslinuxdrive.c

```diff
@@ -40,6 +40,7 @@
 #include <mntent.h>
 
 #include <glib/gstdio.h>
+#include <libstoragemgmt/libstoragemgmt.h>
```

Comment:
I'm not quite sure about this. I understand the fact that this functionality is for local drives, but it's a specific thing for HW that's not commonly used on laptops, workstations, etc., is it? Having this (and/or the better version of ``set_rotation_rate``) in the daemon itself introduces a dependency on LSM for the daemon itself. Which is quite different from having an option to just not build and package the LSM module. The enhancements to the ``set_rotation_rate`` function can be solved by ``#ifdef HAVE_LSM`` or something like that, but a change in the interface cannot be optional like that.